eth_utils/curried/__init__.py

```diff
@@ -102,6 +113,13 @@ def apply_formatter_if(
     ...
 
 
+@overload
+def apply_formatter_if(
+    condition: Callable[..., bool], formatter: Dict[Any, Any]
```

Comment:
I think a comment is warranted here. Why is it possible for the formatter to be a `Dict`?

Reply:
Oops, not sure what I was "fixing" but I don't need it. Nice catch!